lib/Scvmm/miq_scvmm_vm_ssa_info.rb

```diff
@@ -0,0 +1,67 @@
+
+require 'miq_winrm'
+require 'miq_scvmm_parse_powershell'
+
+class MiqScvmmVmSSAInfo
+  attr_reader :vhd, :hostname, :checkpoints, :connection, :vhd_type
+  def initialize(provider, user, pass, port = nil)
+    @checkpoints = []
+    @vhd         = nil
+    @hostname    = nil
+    @vhd_type    = nil
+    @winrm       = MiqWinRM.new
+    winrmport    = port.nil? ? 5985 : port
+    options      = {:port     => winrmport,
+                    :user     => user,
+                    :pass     => pass,
+                    :hostname => provider
+                   }
+
+    @connection  = @winrm.connect(options)
+  end
+
+  def vm_host(vm_name)
+    host_script = <<-host_eol
```

Comment:
Can you use all caps for these heredocs?  They tend to stand out better that way.  In addition, should they be left aligned?  Otherwise this send the actual leading whitespace, and I'm not sure that's intended.